.github/workflows/ci-workflow.yml

```diff
@@ -106,6 +106,10 @@ jobs:
         with:
           working-directory: demo
 
+      # Run rubocop against demo app code
+      - run: bundle exec rubocop --format progress --format github
+        working-directory: demo
```

Comment:
We weren't running this against the demo app code so there were a few lint warnings that were missed. Adds this and then autofixes the warnings.